src/base.js

```diff
@@ -330,17 +330,17 @@ VRDisplay.prototype.addFullscreenListeners_ = function(element, changeHandler, e
   this.fullscreenErrorHandler_ = errorHandler;
 
   if (changeHandler) {
-    element.addEventListener('fullscreenchange', changeHandler, false);
-    element.addEventListener('webkitfullscreenchange', changeHandler, false);
-    document.addEventListener('mozfullscreenchange', changeHandler, false);
-    element.addEventListener('msfullscreenchange', changeHandler, false);
+    if (document.fullscreenEnabled) element.addEventListener('fullscreenchange', changeHandler, false);
+	  else if (document.webkitFullscreenEnabled) element.addEventListener('webkitfullscreenchange', changeHandler, false);
+	  else if (document.mozFullscreenEnabled) document.addEventListener('mozfullscreenchange', changeHandler, false);
```

Comment:
typo here: should be `document.mozFullScreenEnabled` (notice the capitalised `S` in `Screen`)